packages/effector-solid/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "effector-solid",
+  "version": "1.0.0",
```

Comment:
Should we release it as `1.0.0`?

I suggest to set version, that would allow breaking changes after first feedback from users, since current bindings are more of a port of react bindings, than specifically designed for solid 🤔

Reply:
I suggest to set version complement to another bindings. Like `0.22.0` to upgrade after release to `22.0.0` or `23.0.0`

Reply:
True, will be more predictable